lib/nerves_hub_link/socket.ex

```diff
@@ -485,8 +518,8 @@ defmodule NervesHubLink.Socket do
     disconnect(socket)
   end
 
-  defp schedule_network_availability_check(delay \\ 100) do
-    Process.send_after(self(), :connect_check_network_availability, delay)
+  defp schedule_network_and_time_check(delay \\ 100) do
```

Comment:
```suggestion
  defp schedule_preconnect_checks(delay \\ 100) do
```
Maybe we keep this slightly generic to give an entry point running all the checks we need? (At your rate, there might be more coming 🤣)

Reply:
hahaha, I like your naming, and I promise not to add more (for a while)